.travis.yml

```diff
@@ -16,8 +16,9 @@ install:
   - unset _JAVA_OPTIONS
 
   # check if cloud SDK was installed (/bin exists)
-  # travis auto creates cache directories, so remove it before install
   - if [ ! -d $HOME/google-cloud-sdk/bin ]; then
+  # travis auto creates cache directories, so remove it before install
```

Comment:
I think there's a possibility this may work if we give a 2-space indent, so that "# travis ..." aligns with the "if" above,

Reply:
tried it on the first commit in this PR.

Reply:
https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/748/commits/b1751621771851e43230dd3d7ee4ab562f375294

Reply:
Ah, now I get it. The YAML parser will have all these in a singe line.